### PR TITLE
Systems fpc 2023

### PR DESCRIPTION
**Pull request by robertdijk**
Add system configuration for FPC

### CODE REVIEW

**Comment by robertdijk** [comment]
Oops haha

**Comment by robertdijk** [comment]
It is mostly copied from the last nwerc website

File: content/systems.md [hunk 1]
```diff
@@ -110,25 +111,26 @@ Source files submitted to the Judges will be compiled using the following comman
...
+    - Available as `myjava`
 - For Kotlin: the compiled main class will be executed using the following command:
-  - java `kotlin -Dfile.encoding=UTF-8 -J-XX:+UseSerialGC -J-Xss65536k -J-Xms1572864k -J-Xmx1572864k $@`
-  - Available as `mykotlin`
+    - `kotlin -Dfile.encoding=UTF-8 -J-XX:+UseSerialGC -J-Xss65536k -J-Xms1966080k -J-Xmx1966080k $@`
+    - Available as `mykotlin`
 
 Compilation and execution as described above will take place in a “sandbox” on a dedicated judging machine.
-The judging machine will be as identical as possible to, and at least as powerful as, the machines used by teams.
+The judging machine will be identical to the machines used by teams.
```
Comment (mpsijm) [inline content/systems.md:121]: Just double-checking: will they be identical? I assume we'll set up judgehosts in DW again, not in the cloud, but you never know :stuck_out_tongue:

Comment (robertdijk) [reply content/systems.md:121]: Yep that is the plan 👍

**Review by mpsijm — APPROVED** [review]

**Pull request merged**